src/Support/Utils.php

```diff
@@ -48,8 +48,12 @@ public static function namespaceClassname(string $classCandidate, array $namespa
      *
      * @throws \Nuwave\Lighthouse\Exceptions\DefinitionException
      */
-    public static function constructResolver(string $className, string $methodName): Closure
+    public static function constructResolver(string $className, ?string $methodName): Closure
     {
+        if (null === $methodName) {
```

Comment:
Can we just use a default argument here?

Reply:
Not right now, no. The `getMethodArgumentPart` method returns an array which you expand right away. So everywhere `methodName` will be `null` when passed to the `constructResolver` method. 

We'd have to rework all the code around that to be able to use a default argument.

Reply:
This change wasn't working and wasn't relevant in the end. The original change worked fine for detecting errors.